docs/hotspot-makers/5g-hardware-specification.mdx

```diff
@@ -0,0 +1,211 @@
+---
+id: 5g-hardware-specification
+title: 5G Hotspot Hardware Specification
+pagination_label: 5G Hardware Specification
+sidebar_label: 5G Hardware Specification
+description: Helium Documentation
+image: https://i.imgur.com/bWAbESE.png
+slug: /hotspot-makers/5g-hardware-specification
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl'
+
+Hotspots provide sensor devices access to the Helium network via radio. In addition, some Hotspots
```

Comment:
> Hotspots provide sensor devices access to the Helium network via radio

Wondering if `sensor devices` and `radio` are the right words in this context?

Reply:
going to pull this entire paragraph. It is not 5G hardware specific and the content is covered well in other documents